frontend/public/kubevirt/components/vm/vm-detail.jsx

```diff
@@ -30,31 +32,51 @@ import { Disk } from '../disk';
 import { NodeLink, LoadingInline } from '../../../components/utils';
 import { menuActions } from './menu-actions';
 
-const VmiEvents = ({obj: vm}) => {
+const VmEvents = ({ obj: vm }) => {
   const vmi = {
     kind: VirtualMachineInstanceModel.kind,
     metadata: {
-      name: vm.metadata.name,
-      namespace: vm.metadata.namespace,
+      name: getName(vm),
+      namespace: getNamespace(vm),
     },
   };
-  return <ResourceEventStream obj={vmi} />;
+  const launcherPod = {
+    customFilter: ({ kind, namespace, name }) =>
+      kind === PodModel.kind && namespace === getNamespace(vm) && name.startsWith(`${VIRT_LAUNCHER_POD_PREFIX}${getName(vm)}-`),
+  };
+  const importerPod = {
+    customFilter: ({ kind, namespace, name }) => {
+      // importer pod example importer-<diskName>-<vmname>-<generatedId>
+      // note: diskName may contain '-' which means pod name should have at least 4 parts
+      const podNameSplit = name.split('-');
+      const vmName = podNameSplit.length > 3 ? podNameSplit[podNameSplit.length - 2] : null;
+      if (!vmName) {
+        return false;
+      }
+      return kind === PodModel.kind && namespace === getNamespace(vm) && name.startsWith('importer') && vmName === getName(vm);
+    },
+  };
+  const migrationsFilter = {
+    customFilter: ({ kind, namespace, name }) =>
+      kind === VirtualMachineInstanceMigrationModel.kind && namespace === getNamespace(vm) && name === `${getName(vm)}-migration`,
```

Comment:
also better to find by `spec.vmiName` and reuse findVMIMigration (maybe branch that to findActiveVMIMigration)

Reply:
see my comment above, same applies here. `spec` is not available